CODEOWNERS

```diff
@@ -193,19 +179,75 @@ extensions/filters/http/oauth2 @rgs1 @derekargueta @snowp
 /*/extensions/filters/http/set_metadata @aguinet @snowp
 # Formatters
 /*/extensions/formatter/metadata @cpakulski @lizan
-/*/extensions/formatter/req_without_query @dio @tsaarni
 # IP address input matcher
 /*/extensions/matching/input_matchers/ip @aguinet @snowp
 # Key Value store
 /*/extensions/key_value @alyssawilk @ryantheoptimist
-# Stateful session
-/*/extensions/http/stateful_session/cookie @wbpcode @dio
 # DNS Resolver
-/*/extensions/network/dns_resolver/cares @junr03 @yanavlasov
-/*/extensions/network/dns_resolver/apple @junr03 @yanavlasov
 # Config Validators
 /*/extensions/config/validators/minimum_clusters @adisuissa @htuch
 
+# "Unowned extensions (without 1 maintainer or 2 owners)
+# DNS resolution
+/*/extensions/network/dns_resolver/cares @yanavlasov @UNOWNED
+/*/extensions/network/dns_resolver/apple @yanavlasov @UNOWNED
+
+/*/extensions/formatter/req_without_query @UNOWNED @tsaarni
+# Stateful session
+/*/extensions/http/stateful_session/cookie @wbpcode @UNOWNED
+/*/extensions/filters/http/stateful_session @wbpcode @UNOWNED
+# ext_authz
+/*/extensions/filters/common/ext_authz @esmet @gsagula @UNOWNED
+/*/extensions/filters/http/ext_authz @esmet @gsagula @UNOWNED
+/*/extensions/filters/network/ext_authz @esmet @gsagula @UNOWNED
+# compression code
+/*/extensions/filters/http/decompressor @rojkov @UNOWNED
+/*/extensions/filters/http/gzip @gsagula @UNOWNED
+/*/extensions/filters/http/common/compressor @gsagula @rojkov @UNOWNED
+/*/extensions/filters/http/compressor @rojkov @UNOWNED
+/*/extensions/compression/brotli @rojkov @UNOWNED
+/*/extensions/compression/common @UNOWNED @rojkov
+/*/extensions/compression/gzip/decompressor @rojkov @UNOWNED
+/*/extensions/compression/gzip/compressor @rojkov @UNOWNED
+/*/extensions/compression/gzip/common @rojkov @UNOWNED
+# cel
+/*/extensions/access_loggers/filters/cel @UNOWNED @douglas-reid
```

Comment:
I would suggest Google should cover this.

Reply:
I can take cel stuff.